core/store/src/flat/types.rs

```diff
@@ -142,5 +174,24 @@ pub struct FetchingStateStatus {
     pub num_parts: u64,
 }
 
+/// Current step of resharding flat storage - splitting parent.
+#[derive(
+    BorshSerialize, BorshDeserialize, Clone, Debug, PartialEq, Eq, serde::Serialize, ProtocolSchema,
+)]
+pub struct SplittingParentStatus {
```

Comment:
nit: It took me a while to figure out that this is to describe a status of a shard's flat storage rather than the status of resharding. Can you explain that in the comment please?